test/webhook-apicoverage/resourcetree/updateCoverage_test.go

```diff
@@ -29,42 +29,39 @@ func TestSimpleStructValue(t *testing.T) {
 	}
 }
 
-func TestPtrValueAllCovered(t *testing.T) {
-	tree := getTestTree(ptrTypeName, reflect.TypeOf(ptrType{}))
-	tree.UpdateCoverage(reflect.ValueOf(getPtrTypeValueAllCovered()))
-	if err := verifyPtrValueAllCovered(tree.Root); err != nil {
-		t.Fatal(err)
-	}
-}
-
-func TestPtrValueSomeCovered(t *testing.T) {
-	tree := getTestTree(ptrTypeName, reflect.TypeOf(ptrType{}))
-	tree.UpdateCoverage(reflect.ValueOf(getPtrTypeValueSomeCovered()))
-	if err := verifyPtrValueSomeCovered(tree.Root); err != nil {
-		t.Fatal(err)
-	}
-}
-
-func TestArrValueAllCovered(t *testing.T) {
-	tree := getTestTree(arrayTypeName, reflect.TypeOf(arrayType{}))
-	tree.UpdateCoverage(reflect.ValueOf(getArrValueAllCovered()))
-	if err := verifyArryValueAllCovered(tree.Root); err != nil {
-		t.Fatal(err)
+func TestUpdateCoverage(t *testing.T) {
```

Comment:
@mattmoor , there is another test above `TestSimpleStructValue` which I'm not sure how to integrate in this table, as it's `f` take 2 arguments instead of 1. Any idea how to make it work?

Reply:
They don't all have to be in the table, if the same kernel doesn't apply to all the tests.

Reply:
sg, thanks